### PR TITLE
[INF] Localize network services and drivers strings

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

_ Net TCP/IP strings are not localized, therefore some parts of Network Properties are not available for translation

## Proposed changes

_ Localization of display names and descriptions
_ First implementation in English and French

image

### CODE REVIEW

File: media/inf/nettcpip.inf [hunk 1]
```diff
@@ -295,10 +295,24 @@ HKR,"Parameters","ServiceDll",0x00020000,"%SystemRoot%\system32\dnsrslvr.dll"
...
 [Strings.0a]
 ReactOS = "Equipo de ReactOS"
 
+[Strings.040C]
+ReactOS = "Equipe ReactOS"
+MS_TCPIP.DisplayName = "Protocole Internet Version 4 (TCP/IPv4)"
+DNSCLIENT_NAME = "Client DNS" 
+DNSCLIENT_DESC = "Service de mise en case de requêtes DNS locales" 
+DHCPCLIENT_NAME = "Client DHCP" 
+DHCPCLIENT_DESC = "Obtient les paramètres réseaux automatiquement depuis un serveur DHCP"
```
Comment (HBelusca) [inline media/inf/nettcpip.inf:313]:
```suggestion
DHCPCLIENT_DESC = "Obtient les paramètres réseau automatiquement depuis un serveur DHCP"
```

File: media/inf/nettcpip.inf [hunk 2]
```diff
@@ -295,10 +295,24 @@ HKR,"Parameters","ServiceDll",0x00020000,"%SystemRoot%\system32\dnsrslvr.dll"
...
+TCPIP_DEF = "Transmission Control Protocol/Internet Protocol"
 
 [Strings.0a]
 ReactOS = "Equipo de ReactOS"
 
+[Strings.040C]
+ReactOS = "Equipe ReactOS"
+MS_TCPIP.DisplayName = "Protocole Internet Version 4 (TCP/IPv4)"
+DNSCLIENT_NAME = "Client DNS" 
+DNSCLIENT_DESC = "Service de mise en case de requêtes DNS locales"
```
Comment (HBelusca) [inline media/inf/nettcpip.inf:311]:
```suggestion
DNSCLIENT_DESC = "Service de mise en cache des requêtes DNS locales" 
```

File: media/inf/nettcpip.inf [hunk 3]
```diff
@@ -295,10 +295,24 @@ HKR,"Parameters","ServiceDll",0x00020000,"%SystemRoot%\system32\dnsrslvr.dll"
...
+DNSCLIENT_DESC = "Service that caches local DNS queries" 
+DHCPCLIENT_NAME = "DHCP Client" 
+DHCPCLIENT_DESC = "Attempts to obtain network settings automatically from an available DHCP server"
+TCPIP_DEF = "Transmission Control Protocol/Internet Protocol"
 
 [Strings.0a]
 ReactOS = "Equipo de ReactOS"
 
+[Strings.040C]
+ReactOS = "Equipe ReactOS"
```
Comment (HBelusca) [inline media/inf/nettcpip.inf:308]:
```suggestion
ReactOS = "Équipe ReactOS"
```

File: media/inf/nettcpip.inf (hunk 1 above)
Comment (KyleKatarn31) [reply media/inf/nettcpip.inf:313]: OK

File: media/inf/nettcpip.inf (hunk 2 above)
Comment (KyleKatarn31) [reply media/inf/nettcpip.inf:311]: ok

File: media/inf/nettcpip.inf (hunk 3 above)
Comment (KyleKatarn31) [reply media/inf/nettcpip.inf:308]: OK

File: media/inf/nettcpip.inf [hunk 4]
```diff
@@ -295,10 +295,24 @@ HKR,"Parameters","ServiceDll",0x00020000,"%SystemRoot%\system32\dnsrslvr.dll"
...
+DNSCLIENT_NAME = "DNS Client" 
+DNSCLIENT_DESC = "Service that caches local DNS queries" 
+DHCPCLIENT_NAME = "DHCP Client" 
+DHCPCLIENT_DESC = "Attempts to obtain network settings automatically from an available DHCP server"
+TCPIP_DEF = "Transmission Control Protocol/Internet Protocol"
 
 [Strings.0a]
 ReactOS = "Equipo de ReactOS"
 
+[Strings.040C]
```
Comment (binarymaster) [inline media/inf/nettcpip.inf:307]:
```suggestion
[Strings.040c]
```
This one should be lowercase.

**Comment by KyleKatarn31** [comment]
After these 2 approvals, could we go for a merge please ?

**Pull request merged**